lib/xcode.js

```diff
@@ -106,7 +107,7 @@ const getPath = _.memoize(function (timeout = XCRUN_TIMEOUT) {
 
 
 async function getVersionWithoutRetry (timeout = XCRUN_TIMEOUT) {
```

Comment:
is it an internal method?

Reply:
Yes. It is only called within this module, and not exported.